features/camel/src/main/feature/feature.xml

```diff
@@ -16,285 +16,434 @@
           xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
-    <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
-    <repository>mvn:org.apache.karaf.features/enterprise/${karaf.version}/xml/features</repository>
+  <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
 
-    <!--START Camel Features-->
-    <!-- DO NOT EDIT
-        All feature definitions come from 3rd party feature file:
-        <repository>mvn:org.apache.camel.karaf/apache-camel/2.24.2/xml/features</repository>
+  <!--START Camel Features-->
+  <!-- DO NOT EDIT
+      All feature definitions come from 3rd party feature file:
+      <repository>mvn:org.apache.camel.karaf/apache-camel/3.4.4/xml/features</repository>
 
-        All version properties for these Camel features are defined in the platform-app pom.
+      All version properties for these Camel features are defined in the platform-app pom.
```

Comment:
✏️ I know this was here before you changes, but the properties are actually defined in the features pom, not the platform-app pom.